internal/k8stest/k8s_data_helpers.go

```diff
@@ -20,13 +21,19 @@ func HostEndpoint(t *testing.T) string {
 		return "host.docker.internal"
 	}
 
+	host := os.Getenv("HOST_ENDPOINT")
+	if host != "" {
+		return host
+	}
+
 	client, err := docker.NewClientWithOpts(docker.FromEnv)
 	require.NoError(t, err)
 	client.NegotiateAPIVersion(context.Background())
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 	network, err := client.NetworkInspect(ctx, "kind", types.NetworkInspectOptions{})
 	require.NoError(t, err)
+
 	for _, ipam := range network.IPAM.Config {
 		return ipam.Gateway
```

Comment:
In any case I guess it makes sense to check if `ipam.Gateway` is empty before returning?